libs/common-const/src/common.ts

```diff
@@ -166,6 +166,13 @@ export const SWR_OPTIONS = {
   revalidateOnFocus: false,
 }
 
+export const SWR_NO_REFRESH_OPTIONS = {
+  // Cache indefinitely
+  revalidateOnFocus: false,
+  revalidateOnReconnect: false,
+  refreshInterval: 0,
```

Comment:
will this cache also if it fails? or this caches only on successful requests?

Reply:
I assume it would only cache on success, but I have not tested.
Will give it a try.

Reply:
Tested, and confirmed. It does keep trying when there's an error.